src/ios-deploy/ios-deploy.m

```diff
@@ -531,8 +531,15 @@ CFStringRef copy_device_support_path(AMDeviceRef device, CFStringRef suffix) {
     CFRelease(version_parts);
     CFRelease(build);
     CFRelease(deviceClass);
-    if (path == NULL)
+    if (path == NULL) {
+        unsigned int err = 0xe8000070; // kAMDMobileImageMounterMissingImagePath
```

Comment:
Probably better to not use 0xe8000070 / "Could not support development." since it can't be handled differently(Googling that code shows some really odd reasons that code sometimes shows up).

Reply:
Yeah good point. Updated to use custom event indicating device support error.